src/components/seo.tsx

```diff
@@ -84,6 +84,9 @@ const SEO: React.FC<SeoProps> = ({
       )}
       {imageUrl && <meta property="og:image:width" content="300" />}
 
+      {/* -- Xing --*/}
+      <meta property="og:site_name" content={metaDescription} />
```

Comment:
`site_name` should be the `title`. `description` is too long

But still nice that you add the custom description as property, as it's already used on `twitter:desription` for example!